tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -531,7 +530,9 @@ def test_positive_provision_rhev_without_host_group(rhev):
 @pytest.mark.tier3
 @pytest.mark.skip_if_not_set('vlan_networking')
 @pytest.mark.parametrize('setting_update', ['destroy_vm_on_host_delete'], indirect=True)
-def test_positive_provision_rhev_image_based(provisioning, rhev, tear_down, setting_update):
+def test_positive_provision_rhev_image_based_and_disassociate(
```

Comment:
Please include the relevant BZ that's being automated (and check for customer_scenario setting). 

Also please include `parametrized: yes` token in the docblock, since the function is parametrized.